packages/expo-cli/src/commands/eas-build/credentialsSync/action.ts

```diff
@@ -0,0 +1,73 @@
+import CommandError from '../../../CommandError';
+import { Context } from '../../../credentials/context';
+import { updateLocalCredentialsJsonAsync } from '../../../credentials/local';
+import { runCredentialsManager } from '../../../credentials/route';
+import { SetupAndroidBuildCredentialsFromLocal } from '../../../credentials/views/SetupAndroidKeystore';
+import { SetupIosBuildCredentialsFromLocal } from '../../../credentials/views/SetupIosBuildCredentials';
+import prompts from '../../../prompts';
+import { BuildCommandPlatform } from '../types';
+
+interface Options {
+  parent: {
+    nonInteractive?: boolean;
+  };
+}
+
+export default async function credentialsSyncAction(projectDir: string, options: Options) {
+  if (options.parent.nonInteractive) {
+    throw new CommandError('This command is not supported in --non-interactive mode');
+  }
+  const { update, platform } = await prompts([
+    {
+      type: 'select',
+      name: 'update',
+      message: 'What do you want to do?',
+      choices: [
+        {
+          title: 'Update credentials on Expo servers with local credentials.json content',
+          value: 'remote',
+        },
+        { title: 'Update local credentials.json with values from Expo servers', value: 'local' },
+      ],
+    },
+    {
+      type: 'select',
+      name: 'platform',
+      message: 'Do you want to update credentials for both platforms?',
+      choices: [
+        { title: 'Android & iOS', value: BuildCommandPlatform.ALL },
+        { title: 'only Android', value: BuildCommandPlatform.ANDROID },
+        { title: 'only iOS', value: BuildCommandPlatform.IOS },
+      ],
+    },
+  ]);
+  if (update === 'local') {
+    await updateLocalCredentialsJsonAsync(projectDir, platform);
+  } else {
+    await updateRemoteCredentialsAsync(projectDir, platform);
+  }
+}
+
+async function updateRemoteCredentialsAsync(projectDir: string, platform: BuildCommandPlatform) {
```

Comment:
Why is this function defined here and `updateLocalCredentialsJsonAsync` is in a separate file? Please be consistent.